blueprints/native-detector.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const path = require('path');
+
+module.exports = function(blueprint) {
+  blueprint.filesPath = function() {
+    let rootPath = process.env.EMBER_VERSION === 'OCTANE' ? 'native-files' : 'files';
```

Comment:
For reference, we intend to come up with a better "flag" than `process.env.EMBER_VERSION`, but this unlocks actually being able to do the blueprint work while we figure that out.